### DISCUSSION TITLE
Support of icon_links?

### DISCUSSION THREAD

**Discussion by wxrl** [pradyunsg/furo#176] [Ideas] [answered]
It would be nice to have similar support as icon_links from https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#configure-icon-links , which can be placed at the bottom of the side or anywhere else proper. Thanks!

**Comment by pradyunsg** [comment, marked as answer]
Thanks for the suggestion! ^>^

I'd like to not include this style of extensibility in this theme, at the moment. While I agree that this would be nice to have, I don't want to include all the items that all the other themes provide. Notably, this theme doesn't have a top bar or much of a footer, which would be the two main places to put such icons.

https://sphinx-book-theme.readthedocs.io/en/latest/ has recently had a fairly decent stylistic improvement across the board, which might be an alternative worth considering. :)
